draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -352,19 +352,19 @@ In TLS-PSK operation at least the following information from the TLS connection
 ## TLS Session Resumption
 {:#tls_session_resumption}
 
-Session resumption lowers the time and effort required to start a (D)TLS session and increases network responsiveness.
+Session resumption lowers the time and effort required to start a (D)TLS connection and increases network responsiveness.
 This is especially helpful when using short idle timeouts.
 
 RadSec clients and servers SHOULD implement session resumption.
 Implementations supporting session resumption MUST cache data during the initial full handshake, sufficient to allow authorization decisions to be made during resumption.
-For RADIUS/(D)TLS servers, this should preferably be done using stateless session resumption as specified in {{!RFC5077}}, to reduce the resource usage for cached sessions.
+For RADIUS/(D)TLS servers, this should preferably be done using stateless session resumption as specified in {{!RFC5077}}, to reduce the resource usage for cached data.
 
-When resuming a (D)TLS session, both client and server MUST re-authorize the connection by using the original, cached data.
+When establishing a (D)TLS connection with session resumption, both client and server MUST re-authorize the connection by using the original, cached data.
 In particular, this includes the X.509 certificate (when using a PKIX trust model) as well as any policies associated with that identity such as restrictions on source IP address.
 The re-authorization MUST give the same result as if a full handshake was performed at the time of resumption.
 
 If cached data cannot be retrieved securely, resumption MUST NOT be done, by either immediately closing the connection or reverting to a full handshake.
-If a resumed session is closed immediately after being established, the RADIUS/(D)TLS client MUST NOT re-attempt session resumption but perform a full TLS handshake instead.
+If a resumed connection is closed immediately after being established, the RADIUS/(D)TLS client MUST NOT re-attempt session resumption but perform a full TLS handshake instead.
```

Comment:
```suggestion
If a connection that used session resumption is closed immediately after being established, the RADIUS/(D)TLS client MUST NOT re-attempt session resumption but perform a full TLS handshake instead.
```

Re @fmauchle 's comment: maybe it's worth adding what we mean by "immediately after", e.g. "no data sent" or "closed after less than 5 seconds"